litmus-portal/frontend/src/components/CommunityTimeSeriesPlot/index.tsx

```diff
@@ -0,0 +1,317 @@
+
+  const dailyOperators = communityData.google.dailyOperatorData;
+
+  const dailyExperiments = communityData.google.dailyExperimentData;
+
+  const monthlyOperators = communityData.google.monthlyOperatorData;
+
+  const monthlyExperiments = communityData.google.monthlyExperimentData;
+
+  const [x, setX] = React.useState([]);
+
+  const [y, setY] = React.useState([]);
+
+  const [y2, setY2] = React.useState([]);
+
+  const [currentPlotType, setPlotType] = React.useState<{ name: string }>({
+    name: 'Growth',
+  });
+
+  const [currentGranularityType, setGranularityType] = React.useState<{
+    name: string;
+  }>({
+    name: 'Monthly',
+  });
+
+  const handleChangeInType = (
+    event: React.ChangeEvent<{ name?: string; value: unknown }>
+  ) => {
+    const name = event.target.name as keyof typeof currentPlotType;
+    if (name !== currentPlotType.name) {
+      setPlotType({
+        ...currentPlotType,
+        [name]: event.target.value as string,
+      });
+    }
+  };
+
+  const handleChangeInGranularity = (
+    event: React.ChangeEvent<{ name?: string; value: unknown }>
+  ) => {
+    const name = event.target.name as keyof typeof currentGranularityType;
+    if (name !== currentGranularityType.name) {
+      setGranularityType({
+        ...currentGranularityType,
+        [name]: event.target.value as string,
+      });
+    }
+  };
+
+  const cumulativeSum = (timeSeriesCounts: any) => {
+    let countSum = ((sum: any) => (value: any) =>
+      (sum = parseInt(sum, 10) + parseInt(value, 10)))(0);
+    let MappedCumulativeSum = timeSeriesCounts.map(countSum);
+    return MappedCumulativeSum;
+  };
+
+  const processData = () => {
+    let Operators: string[] = [];
+    let Experiments: string[] = [];
+    if (currentGranularityType.name === 'Daily') {
+      dailyOperators.forEach((element: any, index: any) => {
+        Operators.push(element.count);
+        Experiments.push(dailyExperiments[index].count);
+      });
+    } else {
+      monthlyOperators.forEach((element: any, index: any) => {
+        Operators.push(element.count);
+        Experiments.push(monthlyExperiments[index].count);
+      });
+    }
+
+    if (currentPlotType.name === 'Growth') {
+      Operators = cumulativeSum(Operators);
+      Experiments = cumulativeSum(Experiments);
+    }
+
+    const rawData: any = [];
+    if (currentGranularityType.name === 'Daily') {
+      dailyOperators.forEach((element: any, index: any) => {
+        rawData.push({
+          date: element.date,
+          operatorInstalls: Operators[index],
+          experimentRuns: Experiments[index],
+        });
+      });
+    } else {
+      monthlyOperators.forEach((element: any, index: any) => {
+        rawData.push({
+          date: element.date,
+          operatorInstalls: Operators[index],
+          experimentRuns: Experiments[index],
+        });
+      });
+    }
+
+    const x: Date[] = [];
```

Comment:
I think this is related to the top one...this can also be an object

Reply:
Modified it.